diff --git a/jmxtrans/target/generated-resources/appassembler/jsw/jmxtrans/bin/jmxtrans b/jmxtrans/target/generated-resources/appassembler/jsw/jmxtrans/bin/jmxtrans
index d73e009..99c0839 100644
--- a/jmxtrans/target/generated-resources/appassembler/jsw/jmxtrans/bin/jmxtrans
+++ b/jmxtrans/target/generated-resources/appassembler/jsw/jmxtrans/bin/jmxtrans
@@ -346,7 +346,7 @@ checkUser() {
 
         # Still want to change users, recurse.  This means that the user will only be
         #  prompted for a password once. Variables shifted by 1
-        su -m $RUN_AS_USER -c "\"$REALPATH\" $2"
+        su -s /bin/bash $RUN_AS_USER -c "\"$REALPATH\" $2"
         RETVAL=$?
 
         # Now that we are the original user again, we may need to clean up the lock file.

