diff --git a/postgis/build/postgis-2.1.5/liblwgeom/lwin_geojson.c b/postgis/build/postgis-2.1.5/liblwgeom/lwin_geojson.c
index 7019626..c493074 100644
--- a/postgis/build/postgis-2.1.5/liblwgeom/lwin_geojson.c
+++ b/postgis/build/postgis-2.1.5/liblwgeom/lwin_geojson.c
@@ -15,17 +15,25 @@
 #include "lwgeom_log.h"
 #include "../postgis_config.h"
 
-#ifdef HAVE_LIBJSON
 
-#include <string.h>
+#if defined(HAVE_LIBJSON) || defined(HAVE_LIBJSON_C) /* --{ */
+
+
+#define JSON_C_VERSION_013 (13 << 8)
+
 
 #ifdef HAVE_LIBJSON_C
 #include <json-c/json.h>
+#if !defined(JSON_C_VERSION_NUM) || JSON_C_VERSION_NUM < JSON_C_VERSION_013
 #include <json-c/json_object_private.h>
+#endif
 #else
 #include <json/json.h>
+#if !defined(JSON_C_VERSION_NUM) || JSON_C_VERSION_NUM < JSON_C_VERSION_013
 #include <json/json_object_private.h>
 #endif
+#endif
+
 
 #ifndef JSON_C_VERSION
 // Adds support for libjson < 0.10
@@ -33,6 +41,9 @@
 #endif
 
 
+#include <string.h>
+
+
 static void geojson_lwerror(char *msg, int error_code)
 {
 	LWDEBUGF(3, "lwgeom_from_geojson ERROR %i", error_code);
